appveyor.yml

```diff
@@ -43,6 +43,7 @@ test_script:
   - npm --version
   - make sanity-checks
   - cmd: npm test
+  - cmd: make installers-all
```

Comment:
No reason, really. Appveyor defaults to `cmd` if that's not set. I'll remove them in this PR.